src/main/java/frc/robot/subsystems/Wrist.java

```diff
@@ -55,20 +43,46 @@ public Wrist(ShuffleboardTab wristTab) {
     m_motor.setNeutralMode(WristConstants.kNeutralMode);
     m_motor.setInverted(WristConstants.kMotorInvert); 
 
-    m_wristTab = wristTab;
-
     // configure the encoder
     m_absEncoder = new DutyCycleEncoder(WristConstants.kAbsEncoderPort);
+    // Cleaner encoder implementation
+    // offset to zero (arm horizontal)
+    m_absEncoder.setPositionOffset(WristConstants.kEncoderOffset);
+    // scale to radians and invert direction
+    m_absEncoder.setDistancePerRotation(-2.0 * Math.PI);
```

Comment:
We should test that these work, I think we had some issues setting the factors like this. On the other hand that might be because we were using getAbsolutePosition() and not getDistance()